create_framework/http-foundation.rst

```diff
@@ -121,6 +121,13 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well.::
```

Comment:
The double `::` "hack" can only be used when the code to highlight is PHP. In this case we need to highlight JSON, so you'll have to use a single `::` and add a `.. code-block::` directive.